backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/public/functions/validate_surcharge_calculation_SQL.sql

```diff
@@ -0,0 +1,22 @@
+DROP FUNCTION IF EXISTS validate_surcharge_calculation_SQL(text, numeric, numeric)
+;
+
+CREATE OR REPLACE FUNCTION validate_surcharge_calculation_SQL
+(
+    p_sql text,
+    p_Target_PriceList_Version_ID numeric,
+    p_Source_M_ProductPrice_ID numeric
+)
+    RETURNS VOID AS
+$BODY$
+BEGIN
+    SET TRANSACTION READ ONLY;
+
+    EXECUTE p_sql
```

Comment:
why not calling `execute_surcharge_calculation_SQL` with some dummy params (e.g. 0)

Reply:
Because `execute_surcharge_calculation_SQL` gets sql via select based on discount_schema_id param